src/routes/signup/+page.svelte

```diff
@@ -0,0 +1,137 @@
+<script lang="ts">
+	import type { User } from "$lib/db";
+	// import { _signup } from "./+page.server";
+	import "./style.css";
+	let username: string;
+	let password: string;
+	let confirmpassword: string;
+
+	async function signupUser(userData: User) {
+        try {
+            const response = await fetch('/signup', {
+                method: 'POST',
+                headers: {
+                    'Content-Type': 'application/json'
+                },
+                body: JSON.stringify(userData)
+            });
+
+            if (!response.ok) {
+                throw new Error('Failed to register user');
+            }
+
+            const data = await response.json();
+            console.log(data);
```

Comment:
ALWAYS delete console.log, debug like this.
NEVER make your code looks like trash